packages/core/src/reporters/EventRecorderReporter.ts

```diff
@@ -70,14 +70,14 @@ export default class EventRecorderReporter implements StrictReporter {
     this.work('onMutantTested', result);
   }
 
-  public onScoreCalculated(score: ScoreResult): void {
-    this.work('onScoreCalculated', score);
-  }
-
   public onMutationTestReportReady(report: mutationTestReportSchema.MutationTestResult): void {
     this.work('onMutationTestReportReady', report);
   }
 
+  public onScoreCalculated(score: ScoreResult): void {
```

Comment:
Does this mean that you'll get the deprecation warning all the time when this reporter is enabled?

Reply:
Yes, you'll get a warning at the end of every run if you have this reporter enabled

Reply:
Yeah, I guess it's not a big deal